src/foundry/prosemirror/schema/schema-definition.d.mts

```diff
@@ -0,0 +1,36 @@
+import { MarkSpec, Node, NodeSpec } from "prosemirror-model";
+
+export default SchemaDefinition;
+
+/**
+ * An abstract interface for a ProseMirror schema definition
+ */
+declare abstract class SchemaDefinition {
+  /**
+   * The HTML tag selector this node is associated with
+   */
+  static tag: string;
+
+  /**
+   * Schema attributes
+   */
+  static get attrs(): Record<string, any>;
+
+  /**
+   * Check if an HTML element is appropriate to represent as this node, and if so, extract its schema attributes.
+   * @param el - The HTML element
+   * @returns Returns false if the HTML element is not appropriate for this schema node, otherwise returns its attributes.
+   */
+  static getAttrs(el: HTMLElement): Record<string, any> | boolean;
+
+  /**
+   * Convert a ProseMirror Node back into an HTML element.
+   * @param node - The ProseMirror node.
+   */
+  static toDOM(node: Node): any[];
```

Comment:
In general `unknown` is usually the better answer to `any`. While `any` has its use in generic contexts, `unknown[]` provides more type safety here.